modules/spring-config/src/main/java/org/mule/config/spring/MuleConfigurationConfigurator.java

```diff
@@ -84,6 +89,22 @@ public Object getObject() throws Exception
         }
     }
 
+    private void determineDefaultProcessingStrategy(DefaultMuleConfiguration defaultConfig)
+    {
+        if (config.getDefaultProcessingStrategy() != null)
+        {
+            defaultConfig.setDefaultProcessingStrategy(config.getDefaultProcessingStrategy());
+        }
+        else
+        {
+            String processingStrategyFromSystemProperty = System.getProperty(MuleProperties.MULE_DEFAULT_PROCESSING_STRATEGY);
```

Comment:
on second thoughts, I think that the implementation I did here is better than that.

First of all, applySystemProperties() is already quite ugly as it is like to add yet another if to it. Secondly, if I do it there, I would have to give DefaultMuleConfiguration some level of knowledge about how to transform the system property string into a ProcessingStrategy (even if that's by delegating into the ProcessingStrategyUtils). On the other hand, as this is implemented now, I can use a PropertyEditor to decouple that.

I'm sticking to the current implementation